src/tools/PrizeTierController/SavePrizeTiersModal/PrizePoolEditsDisplay.tsx

```diff
@@ -0,0 +1,138 @@
+import { BigNumber } from 'ethers'
+
+export const PrizePoolEditsDisplay = (props: {
+  prizePool: PrizePool
+  oldConfig: PrizeTierConfig
+  newConfig: PrizeTierConfig
+  edits: PrizeTierEditsCheck
+  displayRawValues?: boolean
+}) => {
+  const { prizePool, oldConfig, newConfig, edits, displayRawValues } = props
+  const { data: tokens } = usePrizePoolTokens(prizePool)
+
+  const formattedOldConfig = formatPrettyConfig(oldConfig, tokens.token.decimals)
+  const formattedNewConfig = formatPrettyConfig(newConfig, tokens.token.decimals)
+
+  if (edits.edited) {
+    return (
+      <li className='bg-pt-purple-dark p-3 rounded-xl'>
+        <PrizePoolTitle prizePool={prizePool} className='mb-2 pb-2 border-b' />
+        <div className='flex flex-col gap-2'>
+          {edits.bitRangeSize && (
+            <EditedNumberValue
+              name='Bit Range Size'
+              values={[oldConfig.bitRangeSize, newConfig.bitRangeSize]}
+              formattedValues={[formattedOldConfig.bitRangeSize, formattedNewConfig.bitRangeSize]}
+              displayRawValues={displayRawValues}
+            />
+          )}
+          {edits.expiryDuration && (
+            <EditedNumberValue
+              name='Expiry Duration'
+              values={[oldConfig.expiryDuration, newConfig.expiryDuration]}
+              formattedValues={[
+                formattedOldConfig.expiryDuration,
+                formattedNewConfig.expiryDuration
+              ]}
+              displayRawValues={displayRawValues}
+            />
+          )}
+          {edits.maxPicksPerUser && (
+            <EditedNumberValue
+              name='Max Picks Per User'
+              values={[oldConfig.maxPicksPerUser, newConfig.maxPicksPerUser]}
+              formattedValues={[
+                formattedOldConfig.maxPicksPerUser,
+                formattedNewConfig.maxPicksPerUser
+              ]}
+              displayRawValues={displayRawValues}
+            />
+          )}
+          {edits.endTimestampOffset && (
+            <EditedNumberValue
+              name='End Timestamp Offset'
+              values={[oldConfig.endTimestampOffset, newConfig.endTimestampOffset]}
+              formattedValues={[
+                formattedOldConfig.endTimestampOffset,
+                formattedNewConfig.endTimestampOffset
+              ]}
+              displayRawValues={displayRawValues}
+            />
+          )}
+          {edits.prize && (
+            <EditedBigNumberValue
+              name='Prize Amount'
+              values={[oldConfig.prize, newConfig.prize]}
+              formattedValues={[formattedOldConfig.prize, formattedNewConfig.prize]}
+              displayRawValues={displayRawValues}
+            />
+          )}
+          {edits.tiers.map(
+            (tier, i) =>
+              tier && (
+                <EditedNumberValue
+                  name={`Tier ${i + 1}`}
+                  values={[oldConfig.tiers[i], newConfig.tiers[i]]}
```

Comment:
For tiers we should also show the value of the tier, not just the percentage, so users can validate that it matches the input they provided.